test/agent-keys.test.js

```diff
@@ -19,9 +20,9 @@ var ID_DSA_MD5 = 'a6:e6:68:d3:28:2b:0a:a0:12:54:da:c4:c0:22:8d:ba';
 var ID_ECDSA_FP = 'SHA256:ezilZp/ZHJuMF8i9jyMGuxRdFCu4rzGYLQmfSOhrolE';
 var ID_ECDSA_MD5 = '00:74:32:ae:0a:24:3c:7a:e7:07:b8:ee:91:c4:c7:27';
 
-var SIG_RSA_SHA1 = 'parChQDdkj8wFY75IUW/W7KN9q5FFTPYfcAf+W7PmN8yxnRJB884NHYNT' +
```

Comment:
Upon discussion, and reviewing this, I don't think the signature type has anything to do with the key, I think that's all OpenSSH itself. Since OpenSSH has had sha256 support for quite some time, and newer OpenSSH has dropped support for sha1, I think it's fine for the test to be only sha256.